packages/yojson/yojson-1.2.3/descr

```diff
@@ -0,0 +1,9 @@
+Yojson is an optimized parsing and printing library for the JSON format 
```

Comment:
Sorry, I missed that this release is misnamed -- it should be `yojson.1.2.3` and not `yojson-1.2.3`. Any chance of a fixed PR?